distros/spring/src/test/java/com/connexta/transformation/service/TransformControllerTest.java

```diff
@@ -13,12 +13,55 @@
  */
 package com.connexta.transformation.service;
 
-import org.junit.jupiter.api.Test;
+import static org.hamcrest.core.Is.is;
 
+import com.connexta.transformation.rest.models.TransformRequest;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.ResourceAccessException;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(properties = "endpoints.lookupService.url = http://localhost:")
+@ContextConfiguration
 public class TransformControllerTest {
+  private static final String ACCEPT_VERSION = "Accept-Version";
+
+  @LocalServerPort private int port;
+
+  @Autowired private TestRestTemplate restTemplate;
+
+  @Autowired private String lookupServiceUrl;
 
   @Test
-  public void testTransform() {
-    System.out.println("testing transform");
+  public void testAcceptedResponseEntity() {
+    TransformRequest transformRequest = new TransformRequest();
+    TransformController controller =
+        new TransformController(restTemplate.getRestTemplate(), lookupServiceUrl + port);
+    ResponseEntity responseEntity;
```

Comment:
minor nitpick, should we just assign responseEntity here?

Reply:
Absolutely. I'd like to say this is an artifact of something else I was trying, but I'm not sure. Good suggestion 👍